Description: Fix typo preventing the ddns status text from being colored.
Author: Andreas Hasenack <andreas@canonical.com>
Last-Update: 2022-01-25

--- a/src/bin/keactrl/keactrl.in
+++ b/src/bin/keactrl/keactrl.in
@@ -541,7 +541,7 @@
         d2_status=$inactive
         check_running "$(basename -- "${dhcp_ddns_srv}")"
         if [ "${_running}" -eq 1 ]; then
-            d2_status="active"
+            d2_status="$active"
         fi
         printf "DHCP DDNS: %b\n" "${d2_status}"
 
